src/app/zap-templates/zcl/data-model/chip/localization-time-format-cluster.xml

```diff
@@ -30,8 +30,35 @@ may have differing preferences for how dates and times are conveyed. As such, No
 or audibly convey time information need a mechanism by which they can be configured to use a
 user’s preferred format.</description>
     <!-- Base data types -->
+    <attribute side="server" code="0x0000" define="HOUR_FORMAT" type="HourFormatType" writable="true" optional="false">HourFormat</attribute>
```

Comment:
If the struct and the attribute have the same name, the C code can't compile because in C a class is really a struct and then you define another struct with the same name.

Reply:
They're in different namespaces.  Which code is using barewords and not disambiguating?